symphony-bdk-examples/bdk-spring-boot-example/src/main/resources/application.yaml

```diff
@@ -0,0 +1,15 @@
+domain: devx1.symphony.com
```

Comment:
+1 on reusing the same format

Reply:
idea for later, provide properties documentation with https://docs.spring.io/spring-boot/docs/current/reference/html/appendix-configuration-metadata.html
and validation with javax.validation

Reply:
Already in place actually : https://github.com/SymphonyPlatformSolutions/symphony-api-client-java/pull/186/files#diff-ff7ac3e8e0ec4f8c75d855be89102922R201 but I need to rework on the configuration bean